README.md

```diff
@@ -26,11 +26,13 @@ Join [#events-app-beta](https://hubspotdev.slack.com/archives/C011GFF8KNZ) in th
    - Uses the `Events app` module in a page template
    - References the `Events` HubDB table, set up for dynamic pages
 
-7. Registrations are stored in the CRM using the [Forms API](https://developers.hubspot.com/docs/methods/forms/forms_overview). For each event that you create, you'll need to create a form and add that form ID to the HubDB entry. The form fields needed are:
+7. Registrations are stored in the CRM using the [Forms API](https://developers.hubspot.com/docs/methods/forms/forms_overview). You'll need to create a form and store its GUID in your portal's secrets so your functions have access to it. The form fields needed are:
    - First name
    - Last name
    - Email
 
+8. Use `yarn hs secrets add EVENTS_FORM_GUID <form-guid-goes-here>` to add the Event Registration form's id to your portal's secrets.
```

Comment:
Is there a reason that this is preferred over a field in the events app module?

Reply:
I'm definitely open to it! I had thought of the form as more of a developer object created to receive the submissions and associate visitor cookies with contacts, which is why I kept it in the developer setup (as opposed to leaving it to the marketer to select it).

At least presently, I think it's confusing from a marketer point of view to create a form for use with the app, but that _actual_ form you're looking at in the HS forms tool isn't embedded in the app. So if the marketer has an expectation that they can adjust copy / styling / fields on the form and have it reflect in the app, it would lead to some unexpected behavior. 

Maybe this is an indication that we should rethink how the form submission is working and use a hubspot 'native' form via `hbspt.forms.create()` (instead of 'integrating' with one via https://github.com/HubSpot/cms-event-registration/blob/master/src/event.functions/eventSignup.js#L5 ) that's selectable in the module?

Reply:
I think using secrets for something that is not actually a secret is a sign that we should find an alternative. Until we have some sort of app as a concept that includes a UI for managing configuration, I think we should use a module field.